gen_spec.py

```diff
@@ -1,10 +1,9 @@
-import requests
 import sys
-from tempfile import TemporaryDirectory
 from subprocess import call, check_output
-
-from jinja2 import Template
+from tempfile import TemporaryDirectory
+import requests
```

Comment:
There should be one blank line between imports from stdlib and third-party modules.

https://www.python.org/dev/peps/pep-0008/#imports